scripts/release/common/setup.sh

```diff
@@ -44,6 +44,9 @@ then
     exit 1
 fi
 
+cd go-algorand
+COMMIT_HASH=$(git rev-parse "${BRANCH}")
+
```

Comment:
Capture the commit hash which we'll use later in the test utility to check against the `stdout` of `algod -v`.